### DISCUSSION TITLE
发现了一个小bug

### DISCUSSION THREAD

**Discussion by Daniel19491001** [wh201906/SerialTest#28] [General]
Android端点击数据，编码的下拉菜单箭头，然后就会闪退，手机是realme gt neo，系统基于Android 13

除此之外非常好用的app，感谢分享

**Comment by wh201906** [comment]
你使用的是哪个版本呢？

**Reply by Daniel19491001** [reply]
V0.2.2

**Reply by wh201906** [reply]
很奇怪，我在自己的设备上试下来没有这个问题
你会查看Logcat日志吗？闪退的时候应该有崩溃报告，如果能发过来就好了

**Reply by wh201906** [reply]
另外，如果不点击箭头，直接输入编码名，然后按“设置”按钮，会闪退吗？
如果这样做不会闪退的话你可以先用这个方法对付一下。程序本身支持的编码比下拉菜单中列出来的要多，你可以试一下

**Reply by Daniel19491001** [reply]
我不是做软件的，不会抓取日志，然后打开开发者模式，点那个错误报告生成了一份日志，不知道有没有用，不知道怎么发给你

然后输入编码名好像是不会闪退的，我本身不需要换编码模式，只是偶然发现了这个问题，有的时候一点立马闪退，有的时候不是立马闪退，点完下拉按钮能看见内容，但是一滑动就必定闪退，这个好像是一直可以复现的，如果需要录屏的话也可以发一份给你

**Reply by wh201906** [reply]
方便发到我的邮箱吗？能有录屏就更好了，非常感谢！
我的邮箱是wh201906@yandex.com

**Comment by Daniel19491001** [comment]
更新到最新版本之后问题解决了！感谢！
